tox.ini

```diff
@@ -1,2 +1,12 @@
+[tox]
+envlist = py32, py33, py34
+
+[testenv]
+changedir = src
+commands = python -m unittest discover
+
+[testenv:py32]
+deps = mock
```

Comment:
`mock` is not actually used, it would be better to just remove the import from `test_typing`.